plugins/reindex/src/main/java/org/elasticsearch/plugin/reindex/AbstractBulkIndexByScrollRequestBuilder.java

```diff
@@ -0,0 +1,24 @@
+package org.elasticsearch.plugin.reindex;
```

Comment:
I noted that the license check is not yet working in the Gradle build. If you backport this to 2.x you'll notice that with the Maven build. :) (I once did..)